### PR TITLE
Fix attachment filename

### PR DESCRIPTION
**Pull request by Kennyl**
File attachment should be url encode rather than pure utf-8 format when unidecode unavailable

### CODE REVIEW

**Comment by OzzieIsaacs** [comment]
Could you please give me an example of a non working filename, so I can include it in my secret (to be published) testing scripts.

**Comment by Kennyl** [comment]
https://www.w3.org/Protocols/rfc1341/7_3_Message.html
`Non-ASCII text in the headers of an encapsulated message can be specified using the mechanisms described in [RFC-1342].`

I only find similar answer, but not exactly.
http://stackoverflow.com/questions/33914422/how-can-i-get-effect-of-html-into-emails-subject-line

**Comment by Kennyl** [comment]
or just refer logic to route
/opds/download/
no sanitization  using data.name instead of concentrating title+ author

File: cps/web.py [hunk 1]
```diff
@@ -673,9 +673,9 @@ def get_opds_download_link(book_id, format):
     file_name = book.title
     if len(book.authors) > 0:
         file_name = book.authors[0].name + '-' + file_name
-    # file_name = helper.get_valid_filename(file_name)
```
Comment (Kennyl) [inline cps/web.py]: '-' with remove at get_valid_filename

**Comment by Kennyl** [comment]
you may try 
日経新聞電子版(Free, MAX).recipe  
in calibre fetch news as example

**Pull request merged**